contents/customers/ycombinator.md

```diff
@@ -0,0 +1,48 @@
+---
+title: How Y Combinator is building the next generation of startups, with help from PostHog
+customer: Y Combinator
+logo: ../images/customers/ycombinator/yc_logo_b.png
+featuredImage: ../images/customers/ycombinator/featured.svg
+featuredCustomer: true
+industries:
+    - SaaS, Education
+users:
+    - Leadership
+    - Engineering
+    - Product
+toolsUsed:
+    - Cohorts
+    - Autocapture
+    - PostHog Cloud
+    - Insights
+---
+
+Y Combinator is the world’s top startup accelerator, helping to fund, train and guide the next generation of innovative businesses. Since 2005, thousands of successful companies have emerged from Y Combinator, including Dropbox, Airbnb, PagerDuty, Reddit, Amplitude…and PostHog. 
```

Comment:
space after `Amplitude... `

Reply:
```suggestion
Y Combinator is the world’s top startup accelerator, helping to fund, train and guide the next generation of innovative businesses. Since 2005, thousands of successful companies have emerged from Y Combinator, including Dropbox, Airbnb, PagerDuty, Reddit, Amplitude …and PostHog. 
```